File: VB/DevExpress.Calendar.Model/Appointment.Shared.vb
Imports Microsoft.VisualBasic
	Imports System
Namespace DevExpress.Calendar.Model

	Partial Public Class Appointment

		''' <summary>
		''' CompletionCode
		''' </summary>
		Public Enum CompletionCode As Integer
			NotStarted = 0
			InProgress = 1
			Completed = -1
			Canceled = -2
		End Enum

		''' <summary>
		''' ItemCode
		''' </summary>
		Public Enum ItemCode As Integer
			Appointment = 0
			Meeting = 1
			[Event] = 2
			Task = 3
			Project = 4
		End Enum
	End Class
End Namespace

File: CS/DevExpress.Calendar.Model/Appointment.Shared.cs
namespace DevExpress.Calendar.Model {
    using System;
    
    public partial class Appointment {

        /// <summary>
        /// CompletionCode
        /// </summary>
        public enum CompletionCode : int {
            NotStarted = 0,
            InProgress = 1,
            Completed = -1, 
            Canceled = -2,
        }

        /// <summary>
        /// ItemCode
        /// </summary>
        public enum ItemCode : int {
            Appointment = 0,
            Meeting = 1,
            Event = 2,
            Task = 3,
            Project = 4,
        }        
    }
}